### ISSUE TITLE
How to load with the C api?

### ISSUE THREAD

**Issue by Minipada** [fluent/fluent-bit-go#57]
Hi,
I am trying to load the gstdout as a starting point with the C api. What I did:
1. added a wrapper to flb_plugin_proxy_create in flb_lib to export it
2. in my c program, call the wrapped function with the so path
3. load output:
out_ffd_ = flb_output(ctx_, "gstdout", NULL);
flb_output_set(ctx_, out_ffd_, "match", '*', NULL);
flb_output_set(ctx_, out_ffd_, "param", "a,b,c", NULL);

I am using the master branch of fluentbit, and tested to run with the CLI, which works

The prgram crashes instantly, without information...Am I missing something?

**Comment by Minipada** [comment]
So, the problem was caused by the match line.
Somehow, the single quote for the * element causes the crash. I suppose there are some internals where it picks up the string size but expand the star to something else... Anyway, 

TLDR:
**use "*"** for the match
